### PR TITLE
Move Toc to [cpdocs-toc] shortcode

### PR DESCRIPTION
**Pull request by xxsimoxx**
This allow to generate the ToC using [cpdocs-toc] shortcode.
Adds Shortcodes Everywhere plugin to let shortcodes show up even in widgets.

I have not found any widget areas so look I'll annotate the code where I modified it for testing purposes.

### CODE REVIEW

File: wp-content/themes/cpnet-developer/single.php [hunk 1]
```diff
@@ -19,8 +19,8 @@
 
 			<?php //wporg_developer_post_nav(); ?>
 
-		<?php endwhile; // end of the loop. ?>
+		<?php endwhile;?>
 
-		</main><!-- #main -->
+		</main><?php get_sidebar(); ?><!-- #main -->
```
Comment (xxsimoxx) [inline wp-content/themes/cpnet-developer/single.php:22]: This is where I've somehow added code to display a widgets area.

Comment (viktorix) [reply wp-content/themes/cpnet-developer/single.php:22]: Thanks. Yes, it doesn't have sidebars right now. I will add it.

**Review by viktorix — CHANGES_REQUESTED** [review]
Works great. Just remove Shortcodes Everywhere plugin and submit PR. I'll add `add_filter( 'widget_text', 'do_shortcode' );` to enable shortcodes in widgets.

**Review by viktorix — APPROVED** [review]
Looks good.

**Pull request merged**